File: app/src/main/java/com/glsebastiany/bakingapp/repository/model/Step.java
package com.glsebastiany.bakingapp.repository.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

@Parcel
public class Step {

    @SerializedName("id")
    @Expose
    protected int id;

    @SerializedName("shortDescription")
    @Expose
    protected String shortDescription = null;

    @SerializedName("description")
    @Expose
    protected String description = null;

    @SerializedName("videoURL")
    @Expose
    protected String videoURL = null;

    @SerializedName("thumbnailURL")
    @Expose
    protected String thumbnailURL = null;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getShortDescription() {
        return shortDescription;
    }

    public void setShortDescription(String shortDescription) {
        this.shortDescription = shortDescription;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getVideoURL() {
        return videoURL;
    }

    public void setVideoURL(String videoURL) {
        this.videoURL = videoURL;
    }

    public String getThumbnailURL() {
        return thumbnailURL;
    }

    public void setThumbnailURL(String thumbnailURL) {
        this.thumbnailURL = thumbnailURL;
    }
}

File: app/src/main/java/com/glsebastiany/bakingapp/repository/model/Step.kt
package com.glsebastiany.bakingapp.repository.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import org.parceler.Parcel

@Parcel
class Step {

    @SerializedName("id")
    @Expose
    var id: Int? = null

    @SerializedName("shortDescription")
    @Expose
    var shortDescription: String? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @SerializedName("videoURL")
    @Expose
    var videoURL: String? = null

    @SerializedName("thumbnailURL")
    @Expose
    var thumbnailURL: String? = null

}
